app/views/entries/_entry.html.erb

```diff
@@ -0,0 +1,5 @@
+<% if entry.name == nil %>
```

Comment:
If `name` is a String column, it might be better to use `blank?`.

`entry.name == nil` is false for cases when `entry,name` is an empty string (`""`), but `blank?` covers that and is more readable in my opinion.